Backend/CarRentalApp/CarRentalApp/Models/DTOs/Requests/UserRegistrationDTO.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace CarRentalApp.Models.DTOs.Requests
+{
+    public class UserRegistrationDTO
+    {
+        [Required]
+        [EmailAddress]
+        public string Email { get; set; }
```

Comment:
StringLength?

Reply:
I assumed standard attribute constrain length, but when I looked into its implementation I decided to make my own validation attribute